scalyr_agent/third_party/tcollector/collectors/0/procstats.py

```diff
@@ -37,7 +41,8 @@ def open_sysfs_numa_stats():
     """Returns a possibly empty list of opened files."""
     try:
         nodes = os.listdir(NUMADIR)
-    except OSError, (errno, msg):
+    except OSError as xxx_todo_changeme1:
+        (errno, msg) = xxx_todo_changeme1.args
```

Comment:
Use better variable name here.

Reply:
Oh, I completely forgot about this after I recreated this branch for merging into master, sorry.